kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/JobServiceIntegrationTest.java

```diff
@@ -529,4 +529,38 @@ public void testScheduleAndRunJobWithWorkItem() throws Exception {
                      jobRequest.getCommandName());
         assertNotNull(jobRequest.getData().get("workItem"));
     }
+
+    @Test
+    public void testGetDeletedJob() {
+        final long jobId = 1L;
```

Comment:
I guess that this should be updated to represent non-existent job id. Maybe to put there negative value?